__tests__/store/ui.spec.js

```diff
@@ -131,6 +129,58 @@ describe('store/ui', () => {
 			return expect(ui.actions.search({ dispatch, rootGetters }, randomText))
 				.rejects.toThrow('errorNothingFound');
 		});
-	});
 
+		describe('input value contain whitespace', () => {
+			const runBasicWhitespaceTests = (pageName, inputValue, expectedResult) => {
+				it(`returns ${pageName} page`, async () => {
+					// Arrange:
+					if ('mosaic' === pageName)
+						stub(MosaicService, 'getMosaicInfo').returns(Promise.resolve({}));
+					else if('account' === pageName)
+						stub(AccountService, 'getAccountInfo').returns(Promise.resolve({}));
+					else if ('namespace' === pageName)
+						stub(NamespaceService, 'getNamespaceInfo').returns(Promise.resolve({}));
+
+					// Act:
+					await ui.actions.search({ dispatch, rootGetters }, inputValue);
+
+					// Assert:
+					expect(dispatch).toHaveBeenNthCalledWith(1, 'openPage', expectedResult);
+				});
+			};
+
+			// Arrange:
+			const inputValues = [
+				{
+					pageName: 'block',
+					input: ' 10 '
+				},
+				{
+					pageName: 'account',
+					input: ' DC20B243B63246C9E75E4FB5ED236513A005454393E93C8A4CE6EDEE323C2DDB '
+				},
+				{
+					pageName: 'account',
+					input: ' TAR5OQBKR4KSVRVZ3ZBVHNLMBZ4N4Q27WFVMJDI '
+				},
+				{
+					pageName: 'transaction',
+					input: ' 706BBC8F95AF60B22CB38911A645D3BA24DC480FDBE18C197ACCFE0FDE0DC24D'
+				},
+				{
+					pageName: 'mosaic',
+					input: '3A8416DB2D53B6C8 '
+				},
+				{
+					pageName: 'namespace',
+					input: 'symbol . xym '
+				}
+			];
+
+			inputValues.forEach(({input, pageName}) => runBasicWhitespaceTests(pageName, input, {
+				pageName,
+				param: input.replace(/\s/g, '')
```

Comment:
consider adding expected to last object and then

param: expected ? expected : input

Reply:
> consider adding expected to last object and then

I am confused about this.

the object below is considered as `expectedResult`
```json
{
    pageName,
    param: input.replace(/\s/g, '')
}
```

> param: expected ? expected : input

why do we need this?